src/init.cpp

```diff
@@ -698,11 +698,17 @@ static void ThreadImport(std::vector<fs::path> vImportFiles)
     }
 
     // scan for better chains in the block chain database, that are not yet connected in the active best chain
```

Comment:
nit: That comment is still relevant to the code below but because of the empty line it seems detached.